test/user.test.js

```diff
@@ -6,16 +6,62 @@ const UserModel = require('../dataLayer/model/user.js');
 
 /* eslint-disable no-undef */
 beforeAll(async () => {
-  /* eslint-enable no-undef */
   await mongoose.connect(global.__MONGO_URI__);
 });
 
-/* eslint-disable no-undef */
 afterAll(async () => {
-  /* eslint-enable no-undef */
   await mongoose.disconnect();
 });
 
+// language=GraphQL
+const query = `
+    query { 
+        users {
+            name
+            email
+        }
+    }
+  `;
+
+const rootValue = {};
+
+it(
+  'should return null users and authentication error ' + ' without a token',
```

Comment:
This is hilarious: if I remove splitting of the string, ie: `should return null users and authentication error 
 without a token` running of `yarn format` will have the following result:

'prettier' will pull line 29 and 30 up to line 28 so it's 

```js
it('should return null users and authentication error without a token', async() => {
```
 and next `yarn` lint will scream that line length is longer than 80 😂

If anyone here knows how one is supposed to fix this in a less ugly way: pray tell.

Reply:
`prettier` takes this out of our hands, not leading us down the path of 'how to trick prettier to write the code I like'! :grinning:  :grinning: 

You could try shortening the test message....